common/lib/protocol-definitions/CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # @fluidframework/protocol-definitions Changelog
 
+## [3.2.0](https://github.com/microsoft/FluidFramework/releases/tag/protocol-definitions_v3.2.0)
+
+### Added new optional targetClientId property to ISequencedClient
+
+`ISequencedClient` has a new optional property called `targetClientId` which is the client ID of the singular client the
+signal is being (or has been) sent to.
+
+([#19555](https://github.com/microsoft/FluidFramework/issues/19555)) [af2d678db1](https://github.com/microsoft/FluidFramework/commits/af2d678db1216475a444bd35354c8284e6729973)
+
+### Added new optional omitted property to ISnapshotTree
+
+`ISnapshotTree` has a new optional property, `omitted`. This property will be used by the service to communicate that it
+omitted the blobs contents in the snapshot.
+
+([#19555](https://github.com/microsoft/FluidFramework/issues/19676)) [198235d46d](https://github.com/microsoft/FluidFramework/commits/af2d678db1216475a444bd35354c8284e6729973)
+
+### Added new optional property groupId to ISnapshotTree, ISummaryTree, and ITree
+
+`ISnapshotTree`, `ISummaryTree`, and `ITree` have a new optional property, `groupId`. This `groupId` will be used to
+fetch the missing parts of the snapsot from the service when they are required. When the server returns the snapshot to
+the client and some tree is missing in that snapshot, then this `groupId` will be used to request the missing contents
+from the service.
```

Comment:
Maybe refer to https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/README.md#data-virtualization-for-datastores-work-in-progress ?

To be honest, I'm not sure if we want to communicate here capabilities of protocol (that are being added, but product is not yet in a position to use), or runtime feature (that is there, and eventually will work as described, but noop for now).

The message for service authors - they can implement it if they want / there are scenarios that would benefit from it, but our expectation (for now) is that it will be only implemented in ODSP for now (and there is absolutely no requirement for anybody to implement it - it's opt-in)

Reply:
Thanks - I added a little commentary, added a link to the README you mentioned, and removed the details from the groupId description (point to the docs instead).